src/screens/Documents/Documents.tsx

```diff
@@ -1,13 +1,47 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { TextRegular } from 'assets/fonts/Fonts';
 import { PageContainer } from 'screens/styles';
 import DocContainer from 'components/DocContainer/DocContainer';
+import { getDocList, getAllCases } from 'database/queries';
+import { getCurrentClient } from 'database/auth';
+import { CaseType } from 'types/types';
 
 const UploadScreen = () => {
+  const [caseTypes, setCaseTypes] = useState<CaseType[]>([]); // container headers
+  const [reqDocs, setReqDocs] = useState<string[][]>([]); // container docs
+
+  useEffect(() => {
+    async function loadClientDocs() {
+      const client = await getCurrentClient();
+      if (client !== undefined) {
+        // get all client case types
+        const cases = await getAllCases(client.id);
+        let clientCaseTypes: CaseType[] = cases.map(c => c.type);
+        clientCaseTypes = [...new Set(clientCaseTypes)];
+        clientCaseTypes = Array.from(clientCaseTypes);
+
+        const clientDocs = await Promise.all(
+          clientCaseTypes.map(c => getDocList(c)),
+        );
+
+        setReqDocs(clientDocs);
+        setCaseTypes(Array.from(clientCaseTypes));
```

Comment:
Repeat of line 21? Seems like you can just feed `clientCaseTypes` here without making it an array again

Reply:
I'm not sure if iterating over sets is the same as arrays, but I converted the clientCaseTypes from a set back to an array just to make it simpler.

Reply:
It seems like `clientCaseTypes` is first a set, then you make it into an array on line 21. `clientDocs` is defined by iterating through that array, so it seems like`clientCaseTypes` is  still an array by the time you're using in in line 28?